addons/mail/static/src/components/message_author_prefix/message_author_prefix.xml

```diff
@@ -2,13 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.MessageAuthorPrefix" owl="1">
-        <t t-if="message">
+        <t t-if="messageAuthorPrefixView">
             <span class="o_MessageAuthorPrefix" t-attf-class="{{ className }}" t-ref="root">
-                <t t-if="message.author and message.author === messaging.currentPartner">
-                    <i class="o_MessageAuthorPrefixIcon fa fa-mail-reply"/>You:
-                </t>
-                <t t-elif="thread and message.author !== thread.correspondent">
-                    <t t-esc="message.author.nameOrDisplayName"/>:
+                <t t-if="message">
+                    <t t-if="message.author and message.author === messaging.currentPartner">
+                        <i class="o_MessageAuthorPrefixIcon fa fa-mail-reply"/>You:
+                    </t>
+                    <t t-elif="messageAuthorPrefixView.thread and message.author !== messageAuthorPrefixView.thread.correspondent">
```

Comment:
This condition could be a computed field inside the `messageAuthorPrefixView` ?

Reply:
It could, and also the boolean expression for showing _"You"_.
But all of this is out of scope of PR, which is just to introduce the `MessageAuthorPrefixView` model.
Could have following commits to introduce new fields.